spec/pb_kits/playbook/kits/selectable_list_spec.rb

```diff
@@ -0,0 +1,21 @@
+# frozen_string_literal: true
+
+require_relative "../../../../app/pb_kits/playbook/pb_selectable_list/selectable_list"
+
+RSpec.describe Playbook::PbSelectableList::SelectableList do
+  subject { Playbook::PbSelectableList::SelectableList }
+
+  it { is_expected.to define_partial }
+  it { is_expected.to define_prop(:text) }
+  it { is_expected.to define_enum_prop(:variant)
+                      .with_default("radio")
+                      .with_values("radio", "checkbox")
+  }
+
+  describe "#classname" do
+    it "returns namespaced class name", :aggregate_failures do
+      expect(subject.new({}).classname).to eq "pb_selectable_list_kit"
+      expect(subject.new(classname: "additional_class").classname).to eq "pb_selectable_list_kit additional_class"
+    end
```

Comment:
This is missing a lot of spec coverage.

Reply:
What other test coverage specifically should we add?

Reply:
@thestephenmarshall We added another line to the selectable_list_spec as well as adding a selectable_list_item_spec file with tests in it. Is there any other test coverage you think we should add?